src/app/GitUI/UserControls/RevisionGrid/Graph/RevisionGraphLaneColor.cs

```diff
@@ -28,7 +28,7 @@ static RevisionGraphLaneColor()
             if (branchColors.Length < 2)
             {
                 Trace.WriteLine("At least two graph colors must be configured");
-                branchColors = [Color.Magenta, Color.Cyan];
+                branchColors = [Color.Magenta.AdaptBackColor(), Color.Cyan.AdaptBackColor()];
```

Comment:
ditto

Reply:
Changing, have not tested (the theme colors are changed - as these colors are now themed)